HelpSource/Classes/Event.schelp

```diff
@@ -36,31 +36,91 @@ Returns an empty event with link::#.defaultParentEvent:: as parent.
 method::silent
```

Comment:
now you refer to `IdentityDictionary` for parent and proto, so this needs to go into related (instead of `Dictionary`?)